src/templates/pycontw-2020/contents/en/speaking/cfp.html

```diff
@@ -0,0 +1,98 @@
+{% extends 'contents/_base.html' %}
+
+{% block title %}Call for Proposals {% endblock title %}
+
+{% block meta_description %}The Call for Proposals is now open. PyCon 2020 Taiwan is accepting talks and tutorials!{% endblock meta_description %}
+
+{% block content %}
+{% url 'page' path='speaking/recording' as speaking_recording_url %}
+{% url 'page' path='speaking/talk' as speaking_talk_url %}
+{% url 'page' path='speaking/tutorial' as speaking_tutorial_url %}
+{% url 'page' path='event/open-space' as event_open-space_url %}
+{% url 'page' path='about/code-of-conduct' as about_coc_url %}
+{% url 'signup' as signup_url %}
+{% url 'user_dashboard' as dashboard_url %}
+
+<h1>Call for Proposals </h1>
+
+<p>The Call for Proposals is now open. PyCon 2020 Taiwan is accepting talks and tutorials!</p>
+
+
+<h2>Important Dates(<a href="https://en.wikipedia.org/wiki/Anywhere_on_Earth">AoE</a>)</h2>
```

Comment:
How about `(We use AoE timezone.)` ? @yychen Should we use another style for link text? Otherwise, the user won't notice it's a link

![image](https://user-images.githubusercontent.com/5144808/74603676-0043b880-50f1-11ea-9acb-792096290b0b.png)

Reply:
Sure, content fixed.